cli/config/configuration.toml.in

```diff
@@ -167,6 +167,15 @@ enable_iothreads = @DEFENABLEIOTHREADS@
 # all practical purposes.
 #entropy_source= "@DEFENTROPYSOURCE@"
 
+# Path to OCI hook binaries in the guest rootfs.
+# The agent will search this directory for OCI hooks and add them to the
+# lifecycle of the guest container. Hooks are executed in the runtime namespace
+# of the guest. See the official documentation:
+# https://github.com/opencontainers/runtime-spec/blob/v1.0.1/config.md#posix-platform-hooks
+# Hooks must be stored in a subdirectory of guest_hook_path named after the
+# hook type, e.g. prestart hooks must be in "guest_hook_path/prestart/"
```

Comment:
A few points that I think need to be clarified in this comment:

- When you say, "Path to OCI hook binaries in the guest rootfs" do you mean that this path must already exist in the (implicitly custom) image?
  - If so, what happens if the path isn't valid in the configured image?
  - How would I go about creating such a custom image (add a link to https://github.com/kata-containers/osbuilder).
- What order will hooks be called in?
- Do hooks have to be executable? If not, is this logged?
- If I create `guest_hook_path/prestart/foo/bar/baz.sh`, will it be run? You use the term "search" above which might imply "recursive" unless you specify the precise behaviour :)
- It would be worth noting what is / isn't logged (success, failure, hook name being executed) and where it gets logged.
- It probably wouldn't hurt to list all the sub-directories that will be checked for hooks (since there are only 3 of them).
- I still think the comment needs to be clearer in stating that this is entirely optional and that the "normal" set of OCI hooks will be run (on the host) regardless. It needs to be clearer that these are an "additional" set of hooks I think.